content/en/docs/ops/best-practices/security/index.md

```diff
@@ -42,6 +42,80 @@ For example, in the [authorization for HTTP traffic task](/docs/tasks/security/a
 the authorization policy named `allow-nothing` makes sure all traffic is denied by default.
 From there, other authorization policies allow traffic based on specific conditions.
 
+### Understand path normalization in authorization policy
+
+The enforcement point of authorization policy is the Envoy proxy which is different from the actual resource access point that is
+usually in the backend application. A policy mismatch happens when the Envoy proxy and the backend application interprets the request
+differently.
+
+The mismatch leads to either unexpected rejection or policy bypass. The latter is usually a security incident that needs to be
+fixed immediately, and it's also why we need path normalization in the authorization policy.
+
+For example, you have an authorization policy to reject requests with path `/data/secret`. A request with path `/data//secret` will
+not be rejected because it does not match with the path defined in the authorization policy due to the extra forward slash `/` in the path.
+
+The request goes through and later the backend application returns the same response that it returns for the path `/data/secret`
+because the backend application normalizes the path `/data//secret` to `/data/secret` as it considers the double forward slashes
+`//` is equivalent to a single forward slash `/`.
+
+In this example, the policy enforcement point (Envoy proxy) had a different understanding of the path than to the resource access
+point (backend application). The different understanding caused the mismatch and subsequently the bypass of the authorization policy.
+
+This becomes a complicated problem because of the following factors:
+
+* Lacking of a clear standard of the normalization.
+
+* Backends and frameworks in different layers have their own special normalization.
+
+* Applications can even have arbitrary normalizations for its own use cases.
+
+Istio authorization policy implements built-in support of various basic normalization options to help you to better address
+the problem:
+
+* Read the [Guideline on configuring the path normalization option](/docs/ops/best-practices/security/#guideline-on-configuring-the-path-normalization-option)
+  to understand which normalization option you should use.
+
+* Read the [Customize your system on path normalization](/docs/ops/best-practices/security/#customize-your-system-on-path-normalization) to
+  understand the detail of each normalization option.
+
+* Read the [Less common normalization configurations](/docs/ops/best-practices/security/#less-common-normalization-configurations) for
```

Comment:
Did this section get renamed "Mitigation for unsupported normalization" in [PR #9917](https://github.com/istio/istio.io/pull/9917)?  Also, do you want to add a bullet item for "Safer Authorization Policy Pattern" from that commit?

Reply:
I was actually just using it as a placeholder. Once the other PR is merged, I will update here and a few other places to refer to the new items.

Reply:
updated.